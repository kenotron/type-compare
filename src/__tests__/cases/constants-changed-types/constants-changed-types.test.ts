import { testHarness } from "../../test-harness";
testHarness("constants-changed-types");