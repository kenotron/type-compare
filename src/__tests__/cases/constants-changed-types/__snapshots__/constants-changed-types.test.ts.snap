// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case "constants-changed-types" 1`] = `
{
  "messages": [
    "Export "a" symbol changed from 5 to "hello"",
  ],
  "minChangeType": "major",
}
`;
